extensions/cache/src/main/resources/benchmark-hotrod-streaming.xml

```diff
@@ -28,7 +28,7 @@
                <!-- Working directory for slave -->
                <home>${env.RG_WORK}/slave${slave.index}</home>
                <!-- Path to server ZIP file -->
-               <server-zip>${env.ISPN_ZIP_PATH}/infinispan-server-9.0.0-SNAPSHOT.zip</server-zip>
+               <server-zip>${env.ISPN_9_ZIP_PATH}/infinispan-server-9.0.0-SNAPSHOT.zip</server-zip>
```

Comment:
I left it there for the user as a hint for which file is expected, I am pretty sure they can figure out just removing it in case they want to.
I will get rid of it if you feel it impacts the usefulness of the examples.